packages/sui-studio/workbench/src/components/Root/index.scss

```diff
@@ -1,15 +1,18 @@
+* {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+    Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
 .Root {
   height: 100vh;
-  overflow-y: auto;
+  overflow: hidden;
 
-  &-iframe {
-    background-color: #fff;
-    border: 1px solid gray;
-    border-radius: 4px;
-    display: block;
-    margin: 10px auto;
+  iframe {
+    border: 0;
     overflow: scroll;
     zoom: 1;
+    height: 100vh;
+    width: 100vw;
```

Comment:
With the new changes if we ever have an iframe in a component it would inherit those styles, would'nt they?

Reply:
The only way to change the styles within an iframe is to inject a style tag from outside. The styles are not inherited (that would be a disaster in too many levels xD).